Makefile

```diff
@@ -16,7 +16,7 @@ BOARD		?= qemu
 KERNEL		?= bzImage
 INITRD		?= initrd.cpio.xz
 BUILD		:= build/$(BOARD)
-
+EXFW_OPT	?=
```

Comment:
Unrelated changes, please split it in two patches

Reply:
It's for adding "--repack" for , here we set it to blank by default, I can moved it to another patch.